content/en/docs/kubernetes/getting-started.md

```diff
@@ -0,0 +1,276 @@
+
+One caveat - the chart doesn't send the data to any backend by default. If you
+want to actually use your data in your favorite backend you'll need to configure
+an exporter yourself.
+
+The following values.yaml is what we'll use
+
+```yaml
+mode: daemonset
+
+presets:
+  # enables the k8sattributesprocessor and adds it to the traces, metrics, and logs pipelines
+  kubernetesAttributes:
+    enabled: true
+  # enables the kubeletstatsreceiver and adds it to the metrics pipelines
+  kubeletMetrics:
+    enabled: true
+  # Enables the filelogreceiver and adds it to the logs pipelines
+  logsCollection:
+    enabled: true
+## The chart only includes the loggingexporter by default
+## If you want to send your data somewhere you need to
+## configure an exporter, such as the otlpexporter
+# config:
+# exporters:
+#   otlp:
+#     endpoint: "<SOME BACKEND>"
+# service:
+#   pipelines:
+#     traces:
+#       exporters: [ otlp ]
+#     metrics:
+#       exporters: [ otlp ]
+#     logs:
+#       exporters: [ otlp ]
+```
+
+To use this values.yaml with the chart, save it to your favorite file location
+and then run the following command to install the chart
+
+```sh
+helm install otel-collector open-telemetry/opentelemetry-collector --values <path where you saved the chart>
+```
+
+You should now have a daemonset installation of the collector running in your
+cluster collecting telemetry from each node!
+
+## Deployment Collector
+
+The next step to collecting Kubernetes telemetry is to deploy a deployment
+instance of the collector to gather telemetry related to the cluster as a whole.
+A deployment with exactly 1 replica is used to ensure we don't produce duplicate
+data.
+
+This instance of the collector will use the following components:
+
+- [Kubernetes Cluster Receiver](../collector/components#kubernetes-cluster-receiver):
+  collects cluster-level metrics and entity events.
+- [Kubernetes Objects Receiver](../collector/components#kubernetes-objects-receiver):
+  collects objects, such as events, from the Kubernetes API server.
+
+Lets break these down.
+
+**Kubernetes Cluster Receiver**
+
+The
+[Kubernetes Cluster Receiver](../collector/components#kubernetes-cluster-receiver)
+is the collector's solution for collecting metrics about the state of the
+cluster as a whole. This receiver can gather metrics about node conditions, pod
+phases, container restarts, available and desired deployments, and more.
+
+**Kubernetes Objects Receiver**
+
+The
+[Kubernetes Objects Receiver](../collector/components#kubernetes-objects-receiver)
+is the collector's solution for collecting Kubernetes objects as logs. Although
+any object can be collected, a common and important use case is to collect
+Kubernetes events.
+
+---
+
+The OpenTelemetry Collector Helm chart makes configuring all of these components
+in a deployment installation of the collector easy. It will also take care of
+all of the Kubernetes-specific details such as RBAC, and mounts.
```

Comment:
If it was "easy", there would be a button, so I am suggesting a different phrasing below. 

```suggestion
The OpenTelemetry Collector Helm chart streamlines the configuration for all of these components
in a deployment installation of the Collector. It will also take care of
all of the Kubernetes-specific details, such as RBAC and mounts.
```